src/generic/exports.jl

```diff
@@ -129,5 +129,6 @@ export truncate
 export truncate!
 export unit
 export upscale
+export vector_space_dim
```

Comment:
THis should be moved to `src/exports.jl`.

Then add `function vector_space_dim end` near to where you defined `krull_dim`.

Finally modify `src/generic/imports.jl` by adding a line `import ..AbstractAlgebra: vector_space_dim` (and perhaps also for `krull_dim` and `is_noetherian` as we may wish to define those for at least free modules and direct sum modules?